ACKategoriesCore/RandomExtensions/Int+Random.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+public extension Int {
+    /// - Parameters:
+    ///     - min: Minimal value of returned Int
+    ///     - max: Maximal value of returned Int
+    /// - Returns random Int
+    static func random(min: Int = 20, max: Int = 1000) -> Int {
```

Comment:
❓ what's the added value of this method when we can use `Int.random(in range: Range<Int>)`?

Reply:
None, forgot about this one 😛

Reply:
Well, actually it's nice to have some sensible defaults, so you don't have to input the range everywhere. But I'll rewrite it, so it uses your mentioned method under the hood.